Tests/RegressionTests/run_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Since these tests depends on the Fuzzer, I'd suggest to move the `RegessionTests` directory under `Fuzzers`